src/client/Client.ts

```diff
@@ -22,6 +22,17 @@ import { WatchRestApis } from './watches/WatchRestApis'
 import { UserService } from './UserService'
 import { ProjectService } from './ProjectService'
 
+/**
+ * The default template to use when creating URL to make calls to ops.
+ *
+ * * `origin`: the origin for the client object (no ending slash). For example. 'https://localhost:8080'
+ * * `pathPrefix`: any possible path prefix for the API.
+ * * `project`: the project name.
+ * * `op`: the name of the op.
+ */
+export const DEFAULT_OPS_URL_TEMPLATE =
```

Comment:
So a developer now has the option of passing in a templated string instead of an opsBase object. This makes everything more flexible and doesn't wed anything to projects (with don't exist in Haystack but do in SkySpark).